salt/modules/bigip.py

```diff
@@ -0,0 +1,1723 @@
+# -*- coding: utf-8 -*-
+'''
+An execution module which can manipulate an f5 bigip via iControl REST
+    :maturity:      develop
+    :platform:      f5_bigip_11.6
+'''
+
+# Import python libs
+import requests
```

Comment:
Sorry I missed this one, since requests is an optional salt dep (we had to remove it in 2015.2 because of issues introduced with it in 2014.7) we should gate the import and make a virtual function:
https://github.com/saltstack/salt/blob/develop/salt/modules/etcd_mod.py#L31
https://github.com/saltstack/salt/blob/develop/salt/modules/etcd_mod.py#L52